packages/jmix-front-generator/src/generators/react-typescript/app/answers.ts

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Also default method could be helpful in case `answers` and `questions` are empty.